### PR TITLE
Remove 'Save for later' feature

### PR DESCRIPTION
**Pull request by TBonnin**
## What does this change?
The "Save for later" feature has been disabled for several months now.
This PR is an attempt at removing all code related to Save For Later

I would appreciate if as many people as possible could have a look at it to make sure I have not deleted anything that is still being used especially everything related to CSS 👻  
Let me also know if I forgot something
Thank you

cc @guardian/dotcom-platform 

## What is the value of this and can you measure success?
💀 🔢 🔡 💀 💥 

screen shot 2017-04-11 at 14 44 30

## Tested in CODE?
No

### CODE REVIEW

**Comment by JustinPinner** [comment]
😭

**Comment by paperboyo** [comment]
😭

**Comment by TBonnin** [comment]
Less tears, more reviews. 😜

File: static/src/stylesheets/module/content/_content.global.scss [hunk 1]
```diff
@@ -119,26 +119,6 @@
...
-    .content--liveblog & {
-        @include mq(desktop) {
-            position: relative;
-            left: 0;
-            top: 0;
-            border-top: 1px dotted $neutral-5;
-        }
-    }
-
-    //Todo: Replace to content--gallery when gallery redesign goes live
```
Comment (NataliaLKB) [inline static/src/stylesheets/module/content/_content.global.scss]: 😂 whoops! Clearly I forgot about this one. Glad its getting deleted now.

**Review by NataliaLKB — APPROVED** [review]
🙌

**Review by sndrs — APPROVED** [review]
cor this is lovely! lgtm

**Review by SiAdcock — APPROVED** [review]
🔥  baby 🔥

**Comment by mxdvl** [comment]
Is there any further context on why this was killed beyond it having been turned off for a while?

**Comment by gu-stav** [comment]
I'm afraid I can not help 5 years later. 😢

**Comment by gtrufitt** [comment]
Look for @stephanfowler's kill or keep spreadsheet - IIRC it was because usage was low and didn't bring enough benefit for cost of maintenance and complexity when 3rd party tools existed to do the job better and it was available on the app for 'loyal' readers

**Comment by TBonnin** [comment]
If I remember correctly the feature was rarely used. It was disabled at first and since we got no complain or request to bring it back the code was eventually deleted.

**Comment by stephanfowler** [comment]
All the pointsb already mentioned. Plus as I recall, it relied on (and was
sole client of) a generic "preferences API" in Identity, which was deemed
legacy and not part of the GDPR-motivated overhaul of Identity.

Hello, btw :)

**Comment by gtrufitt** [comment]
Star of Kings?

**Comment by paperboyo** [comment]
Anyone wanting, pls send me your personal gmails, so that I can add you to the Drinking chatroom. Socialising, I meant!

**Comment by sndrs** [comment]
> Is there any further context on why this was killed beyond it having been turned off for a while?

just because you can build things, doesn't mean you should :)

hi everyone! 👋

**Pull request merged**